src/main/java/no/fint/personalmappe/policy/helper/LinkHelper.java

```diff
@@ -0,0 +1,64 @@
+package no.fint.personalmappe.policy.helper;
+
+import no.fint.model.resource.Link;
+import no.fint.model.resource.administrasjon.personal.PersonalmappeResource;
+import no.fint.personalmappe.exception.UnableToGetLink;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.function.Function;
+
+public class LinkHelper {
+
+    private PersonalmappeResource personalmappeResource;
+    private String property;
+    private String identificator;
+
+    public LinkHelper(PersonalmappeResource personalmappeResource) {
+        this.personalmappeResource = personalmappeResource;
+        this.identificator = "systemid";
+    }
+
+    public static Function<PersonalmappeResource, LinkHelper> resource() {
+        return LinkHelper::new;
+    }
+
+    public LinkHelper link(String link) {
+        this.property = link;
+        return this;
+    }
+
+    public LinkHelper id(String identificator) {
+        this.identificator = identificator;
+        return this;
+    }
+
+    public String getLink() {
+        return personalmappeResource
+                .getLinks()
+                .get(property)
```

Comment:
Consider getOrDefault to avoid nullpointer in case of misspelled/non-existent relation.